Scripts/reformat.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+# Save current directory
+DIR=$(pwd)
+
+# Change to the directory passed as argument if any
+if [ $# -eq 1 ]; then
+    cd $1
+fi
+
+# Reformat whole tree.
+# This is run by the reformat target.
+for file in `git ls-files '*.cpp' '*.h'`
+do 
+    python3 Scripts/clang-format.py -i ${file}
```

Comment:
Changing this to
```diff
diff --git a/Scripts/reformat.sh b/Scripts/reformat.sh
index 957d7256c..7c37f3a6e 100755
--- a/Scripts/reformat.sh
+++ b/Scripts/reformat.sh
@@ -10,9 +10,6 @@ fi

 # Reformat whole tree.
 # This is run by the reformat target.
-for file in `git ls-files '*.cpp' '*.h'`
-do
-    python3 Scripts/clang-format.py -i ${file}
-done
+git ls-files -z '*.cpp' '*.h' | xargs -0 -n 1 -P $(nproc) python3 Scripts/clang-format.py -i

-cd $DIR
\ No newline at end of file
+cd $DIR
```
Allows the script to be run threaded which significantly improves execution time.

Reply:
You're right - done.